mucgpt-frontend/src/components/LLMSelector/LLMSelector.module.css

```
@@ -27,27 +27,27 @@
 .buttonContainer:focus-visible {
     outline: 3px solid var(--primary);
 }
 
-.iconRightMargin {
-    margin-right: 8px;
+.iconLeftMargin {
+    margin-left: 8px;
     width: 18px;
     height: 18px;
-    color: var(--primary);
+    color: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)